### PR TITLE
Mawb 19: 1 Create a new repository and push mail sender app

### PR DESCRIPTION
**Pull request by Dinexpod**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (boroda4436) [inline .idea/compiler.xml]: @Dinexpod please remove this file!

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
```
Comment (boroda4436) [inline .idea/vcs.xml]: @Dinexpod please remove this file!

File: src/main/java/mate/academy/mailsender/model/SendMailRequest.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.mailsender.model;
+
+import lombok.Data;
+
+@Data
+public class SendMailRequest {
+
+    private String title;
```
Comment (boroda4436) [inline src/main/java/mate/academy/mailsender/model/SendMailRequest.java]: @Dinexpod Add validations

File: src/main/java/mate/academy/mailsender/config/AppConfig.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.Properties;
+
+@Configuration
+public class AppConfig {
+
+    public static final String MAIL_TRANSPORT_PROTOCOL = "mail.transport.protocol";
+    public static final String MAIL_SMTP_AUTH = "mail.smtp.auth";
+    public static final String MAIL_SMTP_STARTTLS_ENABLE = "mail.smtp.starttls.enable";
+    public static final String MAIL_DEBUG = "mail.debug";
+    public static final String email = "***@gmail.com";
```
Comment (boroda4436) [inline src/main/java/mate/academy/mailsender/config/AppConfig.java]: @Dinexpod make this parameter configurable in application properties

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+    
+</project>
```
Comment (boroda4436) [inline pom.xml]: empty line

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        </dependency>
+        <dependency>
+            <groupId>javax.mail</groupId>
+            <artifactId>mail</artifactId>
+            <version>1.4.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context-support</artifactId>
+            <version>5.1.5.RELEASE</version>
```
Comment (boroda4436) [inline pom.xml]: move version to <properties>

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.mail</groupId>
+            <artifactId>mail</artifactId>
+            <version>1.4.7</version>
```
Comment (boroda4436) [inline pom.xml]: @Dinexpod move this version to property also

File: src/main/java/mate/academy/mailsender/config/AppConfig.java [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.Properties;
+
+@Configuration
+public class AppConfig {
+
+    private static final String MAIL_TRANSPORT_PROTOCOL = "mail.transport.protocol";
+    private static final String MAIL_SMTP_AUTH = "mail.smtp.auth";
+    private static final String MAIL_SMTP_STARTTLS_ENABLE = "mail.smtp.starttls.enable";
+    private static final String MAIL_DEBUG = "mail.debug";
+    private static final String email = "***@gmail.com";
```
Comment (boroda4436) [inline src/main/java/mate/academy/mailsender/config/AppConfig.java]: @Dinexpod use this as a @Value from properties file

File: .idea/compiler.xml (hunk 1 above)
Comment (Dinexpod) [reply .idea/compiler.xml]: +

File: .idea/vcs.xml (hunk 2 above)
Comment (Dinexpod) [reply .idea/vcs.xml]: +

File: src/main/java/mate/academy/mailsender/config/AppConfig.java (hunk 4 above)
Comment (Dinexpod) [reply src/main/java/mate/academy/mailsender/config/AppConfig.java]: +

File: pom.xml (hunk 6 above)
Comment (Dinexpod) [reply pom.xml]: +

File: pom.xml (hunk 5 above)
Comment (Dinexpod) [reply pom.xml]: +

**Pull request closed without merging**